libraries/chain/include/eosio/chain/finality/finalizer.hpp

```diff
@@ -36,7 +36,8 @@ namespace eosio::chain {
       block_ref            last_vote;
       block_ref            lock;
 
-      static constexpr uint64_t magic = 0x5AFE11115AFE1111ull;
+      static constexpr uint64_t magic_unversioned = 0x5AFE11115AFE1111ull;
```

Comment:
I don't see any need to have the extra complexity of supporting the un-versioned file.

Reply:
Why not since people already have unversioned files? And it just a couple lines.